application/src/main/resources/static/js/react-client/src/Pages/faq-screen/question.js

```diff
@@ -10,7 +11,7 @@ const Question = function ({ title, info }) {
         <div className="flex">
           <h3 className="title">{title}</h3>
           <button className={style.btn} onClick={() => setShowInfo(!showInfo)}>
-           
+          {showInfo ? <AiOutlineMinus /> : <AiOutlinePlus />}
```

Comment:
Instead of these react icons, can we just use <p>-</p> and <p>+</p> and style them in CSS as needed